### PR TITLE
build: use go 1.16

### PR DESCRIPTION
**Pull request by chenrui333**
Bump go to 1.16 (relates to Homebrew/homebrew-core#74513)

### CODE REVIEW

**Comment by chenrui333** [comment]
cc @evilsocket

File: go.mod [hunk 1]
```diff
@@ -3,25 +3,24 @@ module github.com/evilsocket/shellz
 go 1.15
```
Comment (evilsocket) [inline go.mod:3]: mmm shouldn't this be 1.16?

Comment (chenrui333) [reply go.mod:3]: yes, this should be go1.16 as well. let me update it.

**Comment by chenrui333** [comment]
cc @evilsocket updated the change

**Pull request merged**